File: go/chapter4/example/app/read/arrayread.go
package read

import "fmt"

func Arrayread() {
	var arr [10]int
	var readCount int = 0
	var readData int

	for {
		fmt.Printf("자연수 입력: ")
		fmt.Scanf("%d", &readData)
		if readData < 1 {
			break
		}
		arr[readCount] = readData
		readCount += 1
	}

	for i := 0; i < readCount; i++ {
		fmt.Printf("%d ", arr[i])
	}
	fmt.Printf("\n")
}

File: python/chapter4/example/app/read/arrayread.py


def Arrayread():
    arr: list = list()
    readCount: int = 0
    readData: int

    while True:
        readData = int(input("자연수 입력: "))
        if readData < 1:
            break

        arr.append(readData)
        readCount += 1

    for i in range(readCount):
        print(f"{arr[i]}", end=" ")
    print("\n")